osi/create-plugin-package.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+# https://docs.aws.amazon.com/opensearch-service/latest/developerguide/custom-plugins.html
+# Note that custom plugins cannot implement ActionPlugin.
+
+FILE_NAME="search-quality-evaluation-plugin-0.0.1.zip"
+
+wget -O ${FILE_NAME} https://github.com/o19s/opensearch-search-quality-evaluation/releases/download/0.0.1/${FILE_NAME}
+
+aws s3 cp ${FILE_NAME} s3://ubi-queries-events-sink/${FILE_NAME} \
+    --profile mtnfog \
+    --region us-east-1
+
+aws opensearch create-package \
```

Comment:
This does not work because it's an `ActionPlugin` but I left this script in here because it does have all of the steps for in case of the future...